controllers/users.js

```diff
@@ -0,0 +1,51 @@
+const User = require('../models/user');
+const User_profile = require('../models/user_profile');
+
+module.exports = {
+index: async(req, res) => {
```

Comment:
here use proper variable names not index

Reply:
done